recompose-parser/src/main/kotlin/recompose/parser/values/Integer.kt

```diff
@@ -0,0 +1,17 @@
+package recompose.parser.values
+
+import org.xmlpull.v1.XmlPullParser
+import recompose.parser.Parser
+
+internal fun XmlPullParser.integer(name: String): Int? {
+    val value = getAttributeValue(null, name)
+
+    try {
+        return when (value) {
+            null -> null
+            else -> Integer.parseInt(value)
+        }
```

Comment:
Since for an integer we will never expect more cases, I guess we could reduce this to:

```Kotlin
return try {
    value?.let { Integer.parseInt(it) }
} // ..
```